user/enterprise/platform-tips.md

```diff
@@ -393,6 +391,32 @@ $ curl -H "Authorization: token XXXXX" https://travis.example.com/api/uptime
 If everything is up and running, it answers with a `HTTP 200 OK`, or in case of failure with a `HTTP 500 Internal Server Error`.
 
 
+## Configuring Backups
+
+This section explains how you integrate Travis CI Enterprise in your backup strategy. Here, we'll talk about two topics:
+
+- The encryption key
```

Comment:
probably link these, although it doesn't make much odds here.

Reply:
Addressed in https://github.com/travis-ci/docs-travis-ci-com/pull/2276/commits/02b82bf917aefc2d9c30980be9662f42c5b3a8e4